### ISSUE TITLE
Detected STILL Activity on debug

### ISSUE THREAD

**Issue by Jinjos** [mauron85/cordova-plugin-background-geolocation#58]
Hello,

I am using Android for this plugin. 

When I activate the bg location, this is the alert I get in the debug, couldn't find anything on this message.

This message is instead of finding the location.

Any idea ?

**Comment by mauron85** [comment]
Still activity means that ActivityRecognition detected STILL (no motion) activity. When device is STILL, location detection is not in use to save power. If you move your device, it should start getting your location. The activity detection interval con be configured ```option.activitiesInterval```. See README.md for more information.

**Comment by Jinjos** [comment]
Thank you !

I will check this now though the alert was the first thing onload, meaning, I never got the 1st latlng.
Is there a way to get the 1st coordinates and then use the no motion feature ?

Thanks

**Comment by mauron85** [comment]
Currently the ANDROID_FUSED_LOCATION don't locate user until some activity is detected.
Your suggestion is actually good idea and I will change it that way.

All that needs to be done is change [detectedActivitiesReceiver](https://github.com/mauron85/cordova-plugin-background-geolocation/blob/master/src/android/FusedLocationService.java#L244) method to something like:

```
    private BroadcastReceiver detectedActivitiesReceiver = new BroadcastReceiver() {
        @Override
        public void onReceive(Context context, Intent intent) {
            ActivityRecognitionResult result = ActivityRecognitionResult.extractResult(intent);
            ArrayList<DetectedActivity> detectedActivities = (ArrayList) result.getProbableActivities();

            //Find the activity with the highest percentage
            lastActivity = getProbableActivity(detectedActivities);

            Log.d(TAG, "MOST LIKELY ACTIVITY: " + getActivityString(lastActivity.getType()) + " " + lastActivity.getConfidence());

            if (lastActivity.getType() == DetectedActivity.STILL) {
                if (config.isDebugging()) {
                    Toast.makeText(context, "Detected STILL Activity", Toast.LENGTH_SHORT).show();
                }
                // startTracking, but only if STILL is first recognize activity (app was just started)
                if (isFirstActivity) {
                  startTracking();
                  isFirstActivity = false;
                }
                // stopTracking();
                // we will delay stop tracking after position is found
            } else {
                if (config.isDebugging()) {
                    Toast.makeText(context, "Detected ACTIVE Activity", Toast.LENGTH_SHORT).show();
                }
                startTracking();
            }
            //else do nothing
        }
    };
```

**Comment by mauron85** [comment]
Please try this updated branch and let me know if this fixed your problem

Remove current version:
```
cordova plugin rm cordova-plugin-mauron85-background-geolocation
```

Install branch fix/58
```
cordova plugin add https://github.com/mauron85/cordova-plugin-background-geolocation#fix/58
```

Build your project
```
cordova build
```

**Comment by mauron85** [comment]
should be fixed in v0.9.5 which is already on npm
